sign.sh

```diff
@@ -162,22 +180,32 @@ function parseArguments() {
   done
 
   ARCHIVE="$1";
+  SIGNING_CERTIFICATE=""
+
+  if [[ $# -ge 2 ]]; then
+    SIGNING_CERTIFICATE="$2";
+  fi
```

Comment:
This can never be true: see lines 160-162 above.

Reply:
I don't follow.

Reply:
The lines (178-180 in the new version) above:
```
  while [[ $# -gt 1 ]] ; do
    shift;
  done
```
ensure that `$#` is at most one.